### DISCUSSION TITLE
hexo deploy 后报错：Could not read from remote repository.

### DISCUSSION THREAD

**Discussion by elarbor** [hexojs/hexo#5442] [Question / Answer]
![image](https://github.com/hexojs/hexo/assets/43592730/321e0d87-cf1f-4029-a56e-a858691a9301)

网上说需要配ssh，我配了。然后部署还是报这个

**Comment by uiolee** [comment]
![info](https://github.com/hexojs/hexo/assets/22849383/c2e035dc-d6f8-4b13-8c05-c71a25323a90)
